### PR TITLE
Update golangci-lint

### PR DESCRIPTION
**Pull request by matejvasek**
# Changes

- :broom: Update golangci-lint

### CODE REVIEW

**Comment by matejvasek** [comment]
@dsimansk @gauron99 PTAL

File: cmd/root.go [hunk 1]
```diff
@@ -303,7 +303,7 @@ func mergeEnvs(envs []fn.Env, envToUpdate *util.OrderedMap, envToRemove []string
 
 	errMsg := fn.ValidateEnvs(envs)
 	if len(errMsg) > 0 {
-		return []fn.Env{}, 0, fmt.Errorf(strings.Join(errMsg, "\n"))
+		return []fn.Env{}, 0, fmt.Errorf("error(s) while validating envs: %s", strings.Join(errMsg, "\n"))
```
[the remark this answers is not in the document]
Comment (matejvasek) [reply cmd/root.go:306]: There may be one or more.

[the remark this answers is not in the document]
Comment (dsimansk) [reply cmd/root.go:306]: IMO something in the realm of covering plural form if present: e.g. 1 error vs. 2 errors

**Pull request merged**